### ISSUE TITLE
Ideas on improvements

### ISSUE THREAD

**Issue by moubi** [moubi/es6-project-template#1]
How can we make it better, more powerful, but still keep it simple.

**Comment by Volzy** [comment]
Can you outline the general requirements?

Should it include:
- Testing?
- Sass / Less?
- Gulp / Grunt?
- Etc?

What's the general idea?

**Comment by moubi** [comment]
Yes, `README` is coming soon. I will try to elaborate with the purpose, rules and direction in there. Stay tuned.
